test/utils.go

```diff
@@ -80,17 +80,6 @@ func getNewWalletAddress(t *testing.T) (address, privateKey string) {
 	return
 }
 
-func sendToAddress(t *testing.T, address string, bsv float64) (txID string) {
```

Comment:
This function will be needed as soon as this is merged: https://github.com/bitcoin-sv/arc/pull/88

Reply:
it's indicated by static check. Not sure we need to keep dead code in repo.

Reply:
This is now merged: https://github.com/bitcoin-sv/arc/pull/88
After a rebase that indication by staticcheck should be gone

Reply:
I'm not sure what to do here as it's staticcheck that complains about dead code. 
If you are fine with keeping dead code and making it harder for us to maintain we can definitely remove staticcheck.

Reply:
No, I'm not saying I want to keep dead code. If you rebase this branch on `main` you'll see that this function will be used and will therefore not be dead code anymore.

Reply:
yup, you are right. added back ;)